### DISCUSSION TITLE
Apps with same name

### DISCUSSION THREAD

**Discussion by Alvtron** [tanujnotes/Olauncher#269] [Ideas]
Hello,

I love Olauncher; especially the type-to-open (spotlight) feature. But when I have multiple apps with the same name, things get less enjoyable.

![Screenshot_20221212_145348_app olauncher](https://user-images.githubusercontent.com/31237428/207063184-42afe9ce-c577-4925-a0cf-8ac527dba776.jpg)

I'd love to be able to differentiate apps with same name. I understand that icons are out of the question, since it breaks the simplicity that this launcher delivers. 

May I suggest, how about allowing apps to be renamed, i. e. nicknamed?

**Comment by jooooscha** [comment]
This has been repeatedly asked, but tanujnotes does not seem to be interrested. Therefore, I take the liberty to lead you to [OlauncherCF](https://github.com/olaunchercf/olaunchercf) which is based on Olauncher but has many more features, including the abillity to change the name of apps.
